packages/framework/src/Twig/ImageExtension.php

```diff
@@ -210,6 +210,14 @@ protected function getImageHtmlByEntityName(array $attributes, $entityName, $add
         $htmlAttributes = $attributes;
         unset($htmlAttributes['type'], $htmlAttributes['size']);
 
+        $useLazyLoading = array_key_exists('lazy', $attributes) ? (bool)$attributes['lazy'] : true;
+        $isAttributeClassExistsAndNotEmpty = array_key_exists('class', $attributes) && $attributes['class'] !== '';
+        $htmlAttributes['class'] = sprintf(
+            '%s%s',
+            $useLazyLoading ? 'lazy' : '',
+            $isAttributeClassExistsAndNotEmpty ? ' ' . $attributes['class'] : ''
```

Comment:
I would prefer to set up space between classes in format pattern, `%s %s` and then use `trim()` to remove unnecessary whitespaces. This will also solve stripping whitespaces from the beginning and end of a string `$attributes['class']`

Reply:
Yes, you are right and I'll think about it in the future. However, this part of code was  remove.